setup.sh

```diff
@@ -5,9 +5,9 @@ echo
 echo "Setting up Dev Env..."
 if [ -d venv ]; then
     echo '-> Using existing virtualenv'
-elif [ `whereis -bq python3.8` ]; then
+elif [ `whereis -b python3.8` ]; then
     virtualenv -p python3.8 venv
-elif [ `whereis -bq python3.9` ]; then
+elif [ `whereis -b python3.9` ]; then
     virtualenv -p python3.9 venv
```

Comment:
I was getting errors that -q option to `whereis` was invalid. Not sure if needed for other platforms.